preprocessing.py

```diff
@@ -56,10 +69,26 @@ def deserializeIndex(filePath):
         all_terms_in_dict = pickle.loads(s)
         return all_terms_in_dict
 
+def sliding_window(size_of_window, docid, dictionary_of_all_terms):
+    with open(docid, 'r') as my_file:
+        data = my_file.read().split()
+    k = len(data) // size_of_window
+    dimension = len(dictionary_of_all_terms)
+    result = []
+    for l in range(0, k):
+        current_vector = sparse.coo_matrix((dimension, 1))
```

Comment:
why do you use coo_matrix instead of lil_matrix for example?
are there some arguments?